presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -115,13 +138,19 @@ public static QueryRunner createJavaQueryRunner(String storageFormat, boolean ad
         return createJavaQueryRunner(Optional.of(getNativeQueryRunnerParameters().dataDirectory), storageFormat, addStorageFormatToPath);
     }
 
+    public static QueryRunner createJavaQueryRunner(String storageFormat, boolean addStorageFormatToPath, boolean createExternalTable)
+            throws Exception
+    {
+        return createJavaQueryRunner(Optional.of(getNativeQueryRunnerParameters().dataDirectory), "sql-standard", storageFormat, addStorageFormatToPath, createExternalTable);
+    }
+
     public static QueryRunner createJavaQueryRunner(Optional<Path> dataDirectory, String storageFormat, boolean addStorageFormatToPath)
             throws Exception
     {
-        return createJavaQueryRunner(dataDirectory, "sql-standard", storageFormat, addStorageFormatToPath);
+        return createJavaQueryRunner(dataDirectory, "sql-standard", storageFormat, addStorageFormatToPath, false);
     }
 
-    public static QueryRunner createJavaQueryRunner(Optional<Path> baseDataDirectory, String security, String storageFormat, boolean addStorageFormatToPath)
+    public static QueryRunner createJavaQueryRunner(Optional<Path> baseDataDirectory, String security, String storageFormat, boolean addStorageFormatToPath, boolean createExternalTable)
```

Comment:
It would be great to make a separate function for createExternalTable that can take more parameters including table name, schema and symlink files so that we can test a database with multiple tables.